test/test_surface_quantity.py

```diff
@@ -43,7 +43,7 @@ def test_surface_flux_export_compute():
     my_export.D = D
 
     # RUN
-    my_export.compute(ds=ds)
+    my_export.compute(n=c, ds=ds)
```

Comment:
Oh right not the good argument. But since n is re defined in the compute function of `SurfaceFlux` that's why the test still pass. We should go back to `my_export.compute(ds=ds)` with no n in argument of the function right ?

Reply:
Yes you are correct we should remove this argument